weboob/banks-all.json

```diff
@@ -365,13 +381,32 @@
         "docType": "Bank",
         "name": "HSBC",
         "uuid": "hsbc",
-        "backend": "weboob"
+        "backend": "weboob",
+        "customFields": [
+            {
+                "name": "secret",
+                "type": "text",
+                "label": "Secret",
+                "labelKey": "settings.secret",
+                "placeholder": "Enter here your secret phrase",
+                "placeholderKey": "settings.secretPlaceholder"
+            }
+        ]
     },
     {
         "docType": "Bank",
         "name": "ING Direct",
         "uuid": "ing",
-        "backend": "weboob"
+        "backend": "weboob",
+        "customFields": [
+            {
+                "name": "birthday",
+                "type": "number",
+                "label": "Birthday",
+                "labelKey": "settings.birthday",
+                "placeholder": "DDMMYYYY"
```

Comment:
We need a placeholderKey here as well, and maybe a translation (JJMMAAAA for fr-fr)